Dining/models.py

```diff
@@ -265,27 +183,20 @@ class DiningWork(models.Model):
 
 
 class DiningEntryUser(DiningEntry, DiningWork):
-    added_by = models.ForeignKey(User, related_name="added_entry_on_dining", on_delete=models.SET_DEFAULT, blank=True,
-                                 default=None, null=True)
-
```

Comment:
Why has this been moved to the normal dining entry. For external dining entries I see no reason why we should want the user being any different than the person who added the external user as adding an external user means that person caries financial responsibility; user and added_by are always the same. Following the logic, if something should change it is that the general user should be renamed to added_by and the DiningEntryUser should have a user field (i.e. the names user and added_by should be reversed.

Reply:
The `created_by` is like metadata. The `user` field is used for determining the kitchen cost and the `created_by` field is used for issues like whoever is allowed to edit the entry. They are both needed for all cases so best to not have them in a subclass. It's simple and explicit to have them separated in this way